test/functional/services/remote/remote.ts

```diff
@@ -144,8 +144,9 @@ export async function RemoteProvider({ getService }: FtrProviderContext) {
   });
 
   lifecycle.cleanup.add(async () => {
-    if (logSubscription) {
-      await new Promise(r => logSubscription!.add(r));
+    // Getting last piece of code coverage before closing browser
+    if (collectCoverage) {
+      await writeCoverage();
```

Comment:
Even after "cleaning" logs subscription and not using `pollForLogEntry$` twice, the process was still stuck at line 148.

I decided to work around it with getting coverage data straight via webdriver: https://github.com/elastic/kibana/pull/53297/files#diff-d0bdf07a0929481211c4a2a1266689a0R59

Reply:
```
       └- ✓ pass  (257ms) "console app console app should resize the editor"
     └-> "after all" hook
   └-> "after all" hook


4 passing (16.2s)

 info writing coverage to /Users/dzmitrylemechko/github/kibana/target/kibana-coverage/functional/2.1576686340275.coverage.json
```